README.md

```diff
@@ -4,13 +4,91 @@ The [Penumbra](https://penumbra.zone/) monorepo for all things web.
+### [Status](https://grpc.testnet.penumbra.zone/): Public info dashboard for Penumbra nodes.
+
+### Packages
+
+**🌘
+[@penumbra-zone/bech32m](https://www.npmjs.com/package/@penumbra-zone/bech32m) 🌑
+[@penumbra-zone/client](https://www.npmjs.com/package/@penumbra-zone/client) 🌑
+[@penumbra-zone/constants](https://www.npmjs.com/package/@penumbra-zone/constants) 🌑
+[@penumbra-zone/crypto](https://www.npmjs.com/package/@penumbra-zone/crypto) 🌑
+[@penumbra-zone/getters](https://www.npmjs.com/package/@penumbra-zone/getters) 🌑
+[@penumbra-zone/keys](https://www.npmjs.com/package/@penumbra-zone/keys) 🌑
+[@penumbra-zone/perspective](https://www.npmjs.com/package/@penumbra-zone/perspective) 🌑
+[@penumbra-zone/protobuf](https://www.npmjs.com/package/@penumbra-zone/protobuf) 🌑
+[@penumbra-zone/services](https://www.npmjs.com/package/@penumbra-zone/services) 🌑
+[@penumbra-zone/services-context](https://www.npmjs.com/package/@penumbra-zone/services-context) 🌑
+[@penumbra-zone/storage](https://www.npmjs.com/package/@penumbra-zone/storage) 🌑
+[@penumbra-zone/transport-chrome](https://www.npmjs.com/package/@penumbra-zone/transport-chrome) 🌑
+[@penumbra-zone/transport-dom](https://www.npmjs.com/package/@penumbra-zone/transport-dom) 🌑
+[@penumbra-zone/types](https://www.npmjs.com/package/@penumbra-zone/types) 🌑
+[@penumbra-zone/wasm](https://www.npmjs.com/package/@penumbra-zone/wasm)
+🌒**
+
+## Documentation
+
+General documentation is available in [docs/README.md](docs/README.md). Package-specific documentation is available in each respective package.
+
+## Getting Started
+
+To start developing the software in this repository, you simply:
+
+- [Install Rust and Cargo](https://doc.rust-lang.org/cargo/getting-started/installation.html) (probably with rustup)
+- [Install Node.js](https://nodejs.org/en/download/package-manager) (at least version 20)
+- [Install pnpm](https://pnpm.io/installation) (probably via corepack)
+- Install Google Chrome
+
+Once you have all these tools, you can
+
+```sh
+git clone https://github.com/penumbra-zone/web
+cd web
+pnpm i
+pnpm dev
+```
+
+You now have a local copy of Minifront available at
+[`https://localhost:5173`](https://localhost:5173) and an unbundled Prax is
+available at `apps/extension/dist/`, ready to be loaded into your testing
+browser.
+
+Minifront will hot-reload. If you're working on the web extension, Chrome
+supports hot-reloading extension pages, but cannot hot-reload the extension
+worker scripts or content scripts.
+
+#### Loading your unbundled build of Prax into Chrome
+
+It's recommended to use a dedicated browser profile for development, not your
+personal profile.
+
+1. Go to the Extensions page [`chrome://extensions`](chrome://extensions)
+2. Enable _Developer Mode_ by clicking the toggle switch at the top right
+3. Click the button _Load unpacked extension_ at the top and locate your cloned
+   repository. Select the extension's build output directory
+   [`apps/extension/dist`](../apps/extension/dist).
```

Comment:
Should probably mention that they'll need to run `pnpm build` or `pnpm dev` for the first time before they can do this.

Reply:
the immediately preceding instructions instruct the user to run dev

Reply:
made it a bit clearer